### PR TITLE
Svgo support for svg compression

### PR DESCRIPTION
**Pull request by jappeace**
Can drastically reduce the size of any svg image on your website by striping out editor tags for example and by minimizing the xml.

### CODE REVIEW

File: optimize_images/Readme.md [hunk 1]
```diff
@@ -2,11 +2,12 @@ Optimize Images Plugin For Pelican
...
 
 This plugin applies lossless compression on JPEG and PNG images, with no
-effect on image quality. It uses [jpegtran][1] and [OptiPNG][2]. It assumes
-that both of these tools are installed on system path.
+effect on image quality. It uses [jpegtran][1], [OptiPNG][2] and [svgo][3]. 
+It assumes that all of these tools are installed on system path.
 
 [1]: http://jpegclub.org/jpegtran/              "jpegtran"
 [2]: http://optipng.sourceforge.net/            "OptiPNG"
+[3]: https://github.com/svg/svgo				"SVGO"
```
Comment (justinmayer) [inline optimize_images/Readme.md]: Suggested changes:
```
This plugin applies lossless compression on JPEG, PNG, and SVG images, with no
effect on image quality, via [jpegtran][], [OptiPNG][], and [svgo][], respectively.
The plugin assumes that all of these tools are installed, with associated
executables available on the system path.

[jpegtran]: http://jpegclub.org/jpegtran/
[OptiPNG]: http://optipng.sourceforge.net/
[svgo]: https://github.com/svg/svgo
```

File: optimize_images/Readme.md [hunk 2]
```diff
@@ -23,4 +24,4 @@ Then use as follows by adding the following to your settings.py:
 Usage
 -----
 The plugin will activate and optimize images upon `finalized` signal of
-pelican.
\ No newline at end of file
+pelican.
```
Comment (justinmayer) [inline optimize_images/Readme.md]: Pelican (i.e., capitalized)

**Comment by jappeace** [comment]
No worries, I actually made this a year ago or so but forgot to make a PR.

**Comment by justinmayer** [comment]
There are a few other suggested text changes that didn't make it over to your latest changes. Perhaps look more closely?

**Comment by jappeace** [comment]
Sorry, I'm blind

**Comment by justinmayer** [comment]
Thanks for the changes!

**Pull request merged**